Signal/src/UIColor+OWS.m

```diff
@@ -54,6 +55,19 @@ + (UIColor *) ows_lightBackgroundColor {
     return [UIColor colorWithRed:242.f/255.f green:242.f/255.f blue:242.f/255.f alpha:1.f];
 }
 
++ (UIColor*) backgroundColorForContact:(NSString*)contactIdentifier {
+    NSArray *colors = @[[UIColor colorWithRed:213.f/255.f green:190.f/255.f blue:112.f/255.f alpha:1.f],
+                        [UIColor colorWithRed:18.f/255.f green:211.f/255.f blue:102.f/255.f alpha:1.f],
+                        [UIColor colorWithRed:32.f/255.f green:144.f/255.f blue:234.f/255.f alpha:1.f],
+                        [UIColor colorWithRed:180.f/255.f green:138.f/255.f blue:255.f/255.f alpha:1.f],
+                        [UIColor colorWithRed:255.f/255.f green:145.f/255.f blue:81.f/255.f alpha:1.f]];
+    NSData *contactData = [contactIdentifier dataUsingEncoding:NSUTF8StringEncoding];
+    NSData *hashData = [Cryptography  computeSHA256:contactData  truncatedToBytes:1];
```

Comment:
Gonna wait before merging. I don't think we're going to go down this road. Picking all these different colors is gonna look really bad but probably @abolishme can show us a mockup of how he wants it to be done instead.